onnxruntime/core/mlas/lib/activate.cpp

```diff
@@ -464,5 +503,10 @@ Return Value:
             MlasActivationKernel<MlasClipActivation>(Activation, Buffer, Bias, M, N, ldc);
             break;
         }
+
+        case MlasHardSigmoidActivation: {
```

Comment:
nit: following existing style, the opening brace should be on new line.

Reply:
Ooops, fixed.